src/pages/workspace/taxes/WorkspaceTaxesSettingsCustomTaxName.tsx

```diff
@@ -48,6 +61,7 @@ function WorkspaceTaxesSettingsCustomTaxName({
                     policyID={policyID}
                     featureName={CONST.POLICY.MORE_FEATURES.ARE_TAXES_ENABLED}
                 >
+                    {' '}
```

Comment:
Why do we need this one?

Reply:
We don't, just my changes didn't push for some reason 😄

Reply:
I believe it's your auto format, I saw it sometimes on my end too 😄